manifests/init.pp

```diff
@@ -203,6 +203,9 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
+# $yum_max_speed::              Maximum bandwidth used per download in kb/sec per worker
```

Comment:
Have a look at https://docs.pulpproject.org/plugins/pulp_rpm/tech-reference/yum-plugins.html?highlight=max_speed#configuration-parameters it says "The maximum download speed in bytes/sec for a task (such as a sync); defaults to None"

As were allowing users to put something like "4 Kb" in maybe something like "The maximum download speed for a Pulp task, such as a sync. (e.g. "4 Kb" or 1Gb")" would be better?